script/brave_license_helper.py

```diff
@@ -21,6 +21,7 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
         # Formerly external Brave code which has moved to brave-core
         # (i.e these are already covered by the Brave Browser license notice).
         os.path.join('brave', 'third_party', 'challenge_bypass_ristretto_ffi'),
+        os.path.join('brave', 'third_party', 'sta_rs'),
```

Comment:
This should instead be a blurb like :
```
        os.path.join('brave', 'third_party', 'sta_rs'): {
            "Name": "sta-rs",
            "URL": "https://github.com/brave/sta-rs",
            "License": "MPL-2.0",
        },
```
inside of `special_cases`: https://github.com/brave/brave-core/blob/943d6b400fbbf81e2a04b3a92eac45af037a52fe/script/brave_license_helper.py#L41-L43